python/nano/src/bigdl/nano/deps/ipex/ipex_quantization_model.py

```diff
@@ -52,6 +55,10 @@ def __init__(self, model: torch.nn.Module, calib_data, q_config=None,
         :param from_load: this will only be set by _load method.
         :param inplace: whether to perform inplace optimization. Default: ``False``.
         :param jit_strict: Whether recording your mutable container types.
+        :param example_kwarg_inputs: keyword arguments of example inputs that will be passed
```

Comment:
may change None, input_sample and use_jit to ` ``None`` `, ` ``input_sample`` ` and ` ``use_jit`` `

Reply:
Done